<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1020</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gbConnection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1001</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QComboBox" name="cbPortName">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="cbBaubrate">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>38400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>125000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>230400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>250000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>460800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>500000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>921600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1000000</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2000000</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbDataBits">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>3</number>
     </property>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>7</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>8</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbStopBits">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cbParity">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>71</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Odd</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Even</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pbConnect">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>50</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Baubrate</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Data bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop bits</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>30</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Parity</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1020</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
